examples/bert/run_glue_finetuning.py

```diff
@@ -153,17 +153,18 @@ def __init__(self, bert_model, hidden_size, num_classes, **kwargs):
             activation="tanh",
             name="pooler",
         )
-        self._logit_layer = tf.keras.layers.Dense(
+        self._probability_layer = tf.keras.layers.Dense(
             num_classes,
-            name="logits",
+            name="probability",
+            activation="softmax",
```

Comment:
Given that the pre training script is doing logit outputs, I think we might want to stay consistent.
https://github.com/keras-team/keras-nlp/blob/master/examples/bert/run_pretraining.py#L119

So stick with logits here, and pass `from_logits=True` to the loss below.

Reply:
sg!